### ISSUE TITLE
Allow marker to be moved

### ISSUE THREAD

**Issue by DanielSiepmann** [DanielSiepmann/PokemonGo-Map#3]
Allow user to move marker to any position on map.
This will disable geolocation and use the location of marker. This will be send to server as `next_loc`.

Allowing user to scan a position he wants to be scanned.

**Comment by DanielSiepmann** [comment]
This should be easy as everything already exists. Just refactor code to a new function that can be called with lat and lng.

https://developers.google.com/maps/documentation/javascript/markers#draggable

```google.maps.event.addListener(marker, 'dragend', function (event) {
    document.getElementById("latbox").value = this.getPosition().lat();
    document.getElementById("lngbox").value = this.getPosition().lng();
});```

http://stackoverflow.com/a/5974362/1888377

**Comment by DanielSiepmann** [comment]
Should be closed through 536a429. But can't test due to offline server.
